src/controllers/usersController.js

```diff
@@ -12,6 +13,18 @@ import { ResponseMaker } from '../utils/responseMaker.js';
 import { saveFileToCloudinary } from '../utils/saveFileToCloudinary.js';
 import { saveFileToUploadDir } from '../utils/saveFileToUploadDir.js';
 
+const getAllUsersController = async (req, res, next) => {
+  const user = await getAllUsers();
+
+  if (!user) return next(HttpError(404, 'User not found'));
+
+  res.status(200).json({
+    status: 200,
+    message: `Successfully found number of all users`,
```

Comment:
'The number of users has been successfully found'

Reply:
Done